ScanbotSDKExamples/NET/ReadyToUseUI.iOS/AppDelegate.cs

```diff
@@ -70,7 +53,25 @@ public override bool FinishedLaunching(UIApplication application, NSDictionary l
             return true;
         }
 
-        string GetDemoStorageBaseDirectory()
+        private static void InitializeScanbotSdk(UIApplication application)
+        {
+            Debug.WriteLine("Scanbot SDK Example: Initializing Scanbot SDK...");
+
+            ScanbotSDKGlobal.SetLoggingEnabled(true);
+            SBSDKUIPageFileStorage.DefaultStorage = new SBSDKUIPageFileStorage(80, new SBSDKStorageLocation(NSUrl.FromFilename(PageStoragePathForExample())));
+            //ScanbotSDKUI.DefaultImageStoreEncrypter = StorageEncryption.FromEncrypter(new SBSDKAESEncrypter("S0m3W3irDL0ngPa$$w0rdino!!!!", SBSDKAESEncrypterMode.SBSDKAESEncrypterModeAES128));
```

Comment:
@scan-mayank, perhaps we should make use of the encryptor for example purposes as it is also used in the Android example. There may need to be a sync with @yurii-scanbot to make sure it is also used consistently in both examples.

Reply:
Using the Native encryptor now.